wikidataQuery.py

```diff
@@ -23,6 +57,8 @@ def searchEntity(entity, string_type):
     params['search'] = entity.rstrip()
     json = requests.get(url,params).json()
     
+    print(entity, '->', json['search'][0]['label'])
```

Comment:
Is this for debugging purposes? If so, can you please remove it?

Reply:
```suggestion
    #print(entity, '->', json['search'][0]['label'])
```

Reply:
```suggestion
```

Reply:
Assuming the verbose branch gets merged first, you can now change this to:
```
if settings.verbose:
    print(entity, '->', json['search'][0]['label'])
```